### DISCUSSION TITLE
Super Mario oddysey adreno 630 SD:845

### DISCUSSION THREAD

**Discussion by SansGamez** [skyline-emu/skyline#1873] [Game Requests]
Yes don't work smo un SD:845 adreno 630

**Comment by parknich082** [comment]
it will not work very good on that + right now its incredibly hard to get into smo let alone on your subpar phone

**Reply by SansGamez** [reply]
Ok

**Comment by jin81790** [comment]
Monster hunter stories wings of ruin on skyline
